examples/IBPD/explicit/ex0/main.cpp

```diff
@@ -271,10 +266,8 @@ class MyIBPDMethod : public IBPDMethod
                 {
                     const double X0_0 = X0[++counter_X0];
                     const double X0_1 = X0[++counter_X0];
-                    const double X0_2 = X0[++counter_X0];
```

Comment:
This looks dangerous.

Note that what is going on here is something like a plain-strain formulation. The current NOSB-PD implementation is always in 3D, so even though this variable is not output, not hitting the iterator seems likely to cause problems in the indexing.

Reply:
You're right. Let me undo this.